server/packaging/ubuntu_xenial/kontena-server/DEBIAN/control

```diff
@@ -2,5 +2,5 @@ Package: kontena-server
 Version: VERSION
 Maintainer: info@kontena.io
 Architecture: all
-Depends: docker-engine (>= 1.10) | docker.io (>= 1.11) | docker-ce | docker-ee, resolvconf
+Depends: docker-engine (>= 1.12) | docker.io (>= 1.12) | docker-ce | docker-ee, resolvconf
```

Comment:
No specific reason to bump the server deps, but I think it just makes sense to keep them the same as the agent deps.